src/librustc_data_structures/bit_set.rs

```diff
@@ -160,7 +160,7 @@ impl<T: Idx> BitSet<T> {
     /// Set `self = self & other` and return true if `self` changed.
     /// (i.e., if any bits were removed).
     pub fn intersect(&mut self, other: &BitSet<T>) -> bool {
-        bitwise(self.words_mut(), other.words(), &Intersect)
+        bitwise(self.words_mut(), other.words(), |a, b| { a & b })
```

Comment:
hmm I guess I can hope this always gets inlined even on debug builds (which I believe was a property of the former architecture here...)